packages/sdk/wallets/wallet-local/src/signing.test.ts

```diff
@@ -142,7 +143,7 @@ describe('Transaction Utils', () => {
     })
   }
 
-  async function verifyLocalSigningInAllPermutations(from: string, to: string): Promise<void> {
+  const verifyLocalSigningInAllPermutations = async (from: string, to: string): Promise<void> => {
```

Comment:
do we like this?
I think there isa downside in that the functions in stacktraces will all be anon.

Reply:
I'm honestly on the fence, I'd be happy to rollback that specific change.